app/users/edit/credits/index/template.hbs

```diff
@@ -1,11 +1,18 @@
 <div class="grid grid--12of12 grid-md--6of12">
   <div class="grid-cell grid-cell-md--12of12 year-select form-inline">
     {{#if this.allowTransfer}}
-      <button type="button" class="btn" {{action (perform this.transfer) on='click'}}>Transfer</button>
+      <button
+        type="button"
+        class="btn"
+        {{action (perform this.transfer) on="click"}}
```

Comment:
`action` => `on` for whole file

Reply:
I dodged this one since it meant additional changed LoC. Sure you want this? :)

Reply:
Since the line is a change already anyway because of formatting, the pr does't get much bigger so i vote in favor of refactor.

Reply:
`done`